packages/ccd-js-gen/README.md

```diff
@@ -314,6 +316,40 @@ const parameter = ...;
 const transactionHash = await MyModule.instantiateMyContract(myModule, transactionMeta, parameter, signer);
 ```
 
+#### function `create<ContractName>ParameterWebWallet`
+
+For each smart contract in module a function for _constructing_ the WebWallet formattet parameter for initializing a new instance is produced. This is to be used with the [`@concordium/wallet-connector`](https://www.npmjs.com/package/@concordium/wallet-connectors) package.
+These are named `create<ContractName>ParameterWebWallet` where `<ContractName>` is the smart contract name in Pascal case.
+
+An example for a smart contract module containing a smart contract named `my-contract`,
+the function becomes `createMyContractParameterWebWallet`.
+
+_This is only generated when the schema contains contract initialization parameter type._
+
+**Parameter**
+
+The function parameter is:
+
+- `parameter` Parameter to provide the smart contract module for the instantiation.
+
+**Returns**: Parameter for initializing a contract instance in the format used by [`@concordium/wallet-connector`](https://www.npmjs.com/package/@concordium/wallet-connectors).
+
+```typescript
+// Wallet connection from `@concordium/wallet-connector`.
+const webWalletConnection: WalletConnection = ...;
+// Parameter to pass the smart contract init-function. The structure depends on the contract.
+const parameter: MyModule.MyContractParameter = ...;
+// Convert the parameter into the format expected by the wallet.
+const walletParameter = MyModule.createMyContractParameterWebWallet(parameter);
```

Comment:
I had to add a type assertion here. Might be useful to mention. The `ccd-js-gen` tool has no access to `TypedSmartContractParameters` type from the `@concordium/wallet-connector` package (since it is not a dependency of this project). So the types are not identical.

```
const walletParameter = MyContract.createMyContractParameterWebWallet(parameter) as unknown as TypedSmartContractParameters;
```